nss-cohort-36/keahua-arboretum-pixel-pals
# Lr create animal menu

# Description

All animal classes should be created and you should be able to create instances of any animal

Fixes #12  (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. `git fetch --all`
2. `git checkout lr-create-animal-menu`
3. In index.py comment out lines 25 and below. 
4. Create an instance of each animal, then for each animal `print(Animal.species)`
5. Run `python index.py` in terminal. This should print each of the animal species. 



# Checklist:
